pandas/tools/merge.py

```diff
@@ -1436,13 +1438,51 @@ def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
     copy : boolean, default True
         If False, do not copy data unnecessarily
 
-    Notes
-    -----
-    The keys, levels, and names arguments are all optional
-
     Returns
     -------
     concatenated : type of objects
+
+    Notes
+    -----
+    The keys, levels, and names arguments are all optional.
+
+    Examples
+    --------
+    Combine two ``Series``.
+
+        >>> import pandas as pd
+        >>> s1 = pd.Series(['a', 'b'])
```

Comment:
no indentation is needed here for the example blocks

(and the 'import pandas as pd' can also be left out -> this is assumed)